.orchestra/config/components/revng-theme.yml

```diff
@@ -0,0 +1,35 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_theme_args():
+repository: revng-vscode-theme
+license: LICENSE.md
+binary_archives: private
+build_dependencies:
+- revng-js-prerequisites
+configure: |
+  mkdir -p "$BUILD_DIR"
+  cd "$BUILD_DIR"
+  cp "$SOURCE_DIR/"package{,-lock}.json .
+  "$ORCHESTRA_DOTDIR"/support/fetch-js-dependencies npm -o ./.cache
+install: |
+  cd "$BUILD_DIR"
+  rsync -aq \
+    --exclude=.git --exclude-from=<(git -C "$SOURCE_DIR" ls-files --exclude-standard -oi --directory) \
+    "$SOURCE_DIR/" .
```

Comment:
```suggestion
  rsync \
    --archive \
    --delete \
    --exclude=.git --exclude-from=<(
      git \
        -C "$SOURCE_DIR" \
        ls-files \
        --exclude-standard \
        --others \
        --ignored \
        --directory) \
    "$SOURCE_DIR/" .
```

Please do not make the build phase quiet, it makes debugging things hard.

Reply:
As said below, rsync generates quite a lot of noise in the build log, I think it's detrimental if something goes wrong and you have to look through "useless" lines

Reply:
OK, `cp -ar` is not verbose by default too.